### DISCUSSION TITLE
self hosted action runner not running any job

### DISCUSSION THREAD

**Discussion by ruchi1803** [community/community#136864] [Actions]
### Select Topic Area

Question

### Body

Hi Team,

I deployed github action controller (ARC) in my AKS cluster and the controller and listener are working fine. If I use the default image from github action `ghcr.io/actions/actions-runner:latest` then the jobs runs fine.
But if  I use my own image, the pods getting created but terminating in 1s and loops goes on until the EphemeralRunner failed for 5 times the default threshold. 
The CI pipeline from github is keep waiting for the agent to spin up but pods are keep crashing and online.
The pod status does show Running for 1s and the terminated. 
Unable to check logs of the pod as well, because the pods terminated by the time if I try to check the logs 

this is from EphemeralRunner
`Pod has failed to start more than 5 times:` 

<img width="747" alt="Screenshot 2024-08-26 at 8 32 54 PM" src="https://github.com/user-attachments/assets/1f9a926c-bf56-4a6b-99dd-18d1e14f4155">
Screenshot of pods loop

**Comment by armyost** [comment]
Can you tell me which stage the pod terminated at.

In the initailizing stage? or Container Creating?

And you can describe the pods with kubectl as well.

**Comment by ruchi1803** [comment]
This is resolved, there was an issue with the run.sh script path in dockefile.
